docs/deployment/help_dpservice-bin.md

```diff
@@ -7,6 +7,7 @@
 | --pf0 | IFNAME | first physical interface (e.g. eth0) |  |
 | --pf1 | IFNAME | second physical interface (e.g. eth1) |  |
 | --pf1-proxy | IFNAME | VF representor to use as a proxy for pf1 packets |  |
+| --pf1-proxy-vf | IFNAME | VF interface of the pf1-proxy VF representor |  |
```

Comment:
Do we have some combinations which do not make sense with the newly introduced flags ?
Like, if I set `pf1-proxy` then I need to set `pf1-proxy-vf `as well ? and If I set multiport-eswitch, I would need` pf1-proxy-* ` switches and if they are not set then I can not operate meaningfully ?

If so, can we document these dependencies ? and enforce during command line argument parsing with meaningful hints returned ?

Also in the documentation, maybe an example dpservice-bin command including all the command line parameters for a simple mpesw operation ?
Or maybe I overlooked it in the documentation ? Otherwise it is not so clear now, how to make mpesw work and which parameters are needed to make it work. At least to me.

These command line arguments are supposed to be generated by `prepare.sh` right ? but still showcasing in an example which parameters needed for dpservice-bin to operate successfully in mpesw mode, would be helpful and how (with which parameters) am I supposed to call prepare.sh if I want to operate in mpesw mode ?